src/editor.js

```diff
@@ -383,6 +383,27 @@ const PreviewPanel = ({
   </div>
 `;
 
+const dispatchUpdatePreview = redispatchAs(g('update-preview'));
+
+const RefreshIcon = () => html`
+  <div class="${n('update-preview-button')}" @click=${dispatchUpdatePreview}>
+    <svg
```

Comment:
Put this in an `svg` :)

Reply:
Actually, you need to nest an `svg` into an `html`:

```js
const RefreshIcon = () => html`
  <div ...>
  ${svg`
    <svg ...>
  `}
  </div>
`;
```

You can simplify by splitting into two template functions:

```js
// (use `RefreshButton`)
const RefreshButton = () => html`
  <div ...>
  ${RefreshIcon()}
  </div>
`;

const RefreshIcon = () => svg`
  <svg ...>
`;
```

Reply:
ohh i see

Reply:
![Screenshot 2019-07-08 at 2 20 08 PM](https://user-images.githubusercontent.com/31637947/60843603-01405f00-a18c-11e9-8538-a1066977f8e8.png)

when i tried the simplified version i got this error but the nesting svg inside the html tag works

Reply:
You'd need to call `RefreshIcon()` inside `RefreshButton` in your example.